addons/mayastor/disable

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -e
+
+source $SNAP/actions/common/utils.sh
+KUBECTL="$SNAP/kubectl --kubeconfig=${SNAP_DATA}/credentials/client.config"
+
+echo "Disabling Mayastor"
+
+# Delete the resources
```

Comment:
It would be better to have a check that makes sure there are no mayastor volumes currently provisioned before disabling it.